target_tools/utils.py

```diff
@@ -173,3 +188,122 @@ def memoized_func(*args, **kwargs):
         return result
 
     return memoized_func
+
+
+def compose_functions(outer, inner):
+    """Compose two functions"""
+    return lambda x: outer(inner(x))
+
+
+def is_string(value):
+    """Checks if value is string"""
+    return isinstance(value, string_types)
+
+
+def is_number(value):
+    """Checks if value is numeric"""
+    try:
+        int(value)
+        return True
+    except (ValueError, TypeError):
+        return False
+
+
+def parse_int(value):
+    """Casts value to integer if possible, otherwise returns None"""
+    try:
+        return int(value)
+    except (ValueError, TypeError):
+        return None
+
+
+def parse_float(value):
+    """Casts value to float if possible, otherwise returns None"""
+    try:
+        return float(value)
+    except (ValueError, TypeError):
+        return None
+
+
+def is_dict(value):
+    """Checks if value is a dict"""
+    return isinstance(value, dict)
+
+
+def is_list(value):
+    """Checks if value is a list"""
+    return isinstance(value, list)
+
+
+def create_uuid():
+    """Create new UUID"""
+    return str(uuid.uuid4())
+
+
+def get_epoch_time(utc_datetime=None):
+    """Get epoch time (seconds) from either passed in UTC datetime or current datetime"""
+    if not utc_datetime:
+        utc_datetime = datetime.datetime.utcnow()
+    return math.ceil((utc_datetime - EPOCH_START).total_seconds())
+
+
+def get_epoch_time_milliseconds(utc_datetime=None):
+    """Get epoch time (milliseconds) from either passed in UTC datetime or current datetime"""
+    epoch_seconds = get_epoch_time(utc_datetime)
+    return epoch_seconds * MILLISECONDS_IN_SECOND
+
+
+def get_timezone_offset():
+    """Get local timezone offset from UTC"""
+    timezone = get_localzone()
+    offset_minutes = timezone.utcoffset(datetime.datetime.now()).total_seconds() // SECONDS_IN_MINUTE
+    return parse_int(offset_minutes)
+
+
+def flatten_list(_list):
+    """Flatten list with nested lists"""
+    if not _list:
+        return []
+    return reduce(operator.add, _list)
+
+
+def remove_empty_values(_dict):
+    """Removes Nonetype dict values"""
+    return {k: v for k, v in list(_dict.items()) if v is not None}
+
+
+def create_visitor(config, visitor_cookie=None, customer_ids=None):
```

Comment:
I see commented out code in `target_python_sdk/__init__.py` calling this function and there are no other usages. If it's not needed, we can remove it from here and also delete that commented out code..